packages/commonwealth/client/scripts/state/ui/thread/index.ts

```diff
@@ -0,0 +1,40 @@
+import { createBoundedUseStore } from 'state/ui/utils';
+import { devtools } from 'zustand/middleware';
+import { createStore } from 'zustand/vanilla';
+
+interface threadCounterStore {
+  totalThreadsInCommunity: number;
+  totalThreadsInCommunityForVoting: number;
+  setTotalThreadsInCommunity: (name: number) => void;
+  setTotalThreadsInCommunityForVoting: (name: number) => void;
+}
+
+export const EXCEPTION_CASE_threadCountersStore =
```

Comment:
I would like this file to disappear completely (take a look on the other comment) but just FYI, this should be `thread.ts` file not `index.ts` file.

Second thing, you have defined setters `setTotalThreadsInCommunity` and `setTotalThreadsInCommunityForVoting` but you are not using it anywhere. You do everywhere `EXCEPTION_CASE_threadCountersStore.setState` which works, but it is wrong. You can take a look on `sidebar` zustand file how setters and getters are defined and how they are used in the codebase.

Reply:
> Second thing, you have defined setters setTotalThreadsInCommunity and setTotalThreadsInCommunityForVoting but you are not using it anywhere. You do everywhere EXCEPTION_CASE_threadCountersStore.setState which works, but it is wrong. You can take a look on sidebar zustand file how setters and getters are defined and how they are used in the codebase.

this is done

> I would like this file to disappear completely (take a look on the other comment) but just FYI, this should be thread.ts file not index.ts file.

I think we resolved it here? @masvelio

Reply:
@jnaviask If we're referring to #3244, then it looks like the `/threads?bulk=true` route is being used to request filtered threads. We can update that route to additionally return whatever count stats need to be rendered to the page.

Reply:
Awesome, thanks @rbennettcw. @mzparacha can you open a ticket describing the ideal output of that route?